style.css

```diff
@@ -1,3 +1,82 @@
+html{
+    width: 375px;
+    height: 100%;
+    padding-left: 20px;
+}
 .header{
-    background-color: aliceblue;
-}
\ No newline at end of file
+    width: 375px;
+    height: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.menu{
+    color: #fff;
+  font-size: 23px;
+  cursor: pointer;
+  display: none;
+}
+
+.menu li {
+    list-style: none;
+    display: inline-block;
+  }
+  .navbar .menu li a {
+    display: block;
+    color: #fff;
+    font-size: 18px;
+    font-weight: 500;
+    margin-left: 25px;
+    transition: color 0.3s ease;
+  }
+.menu li a:hover {
+    color: crimson;
+  }
+.menu li a:hover {
+    color: #fff;
+  }
+
+.hero{
+    width: 375px;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background-image: url(/image/header-shapes\ mobile.png);
+}
```

Comment:
- [ ] Please let's stick to the Figma design as much as possible. Therefore the space between the "Toolbar" section and "Headline" is too much, kindly reduce it.
### Your design :point_down: 
![Screenshot from 2023-05-06 15-25-54](https://user-images.githubusercontent.com/91142432/236634428-dc22e90d-4709-4489-8143-952b6f1cb4f3.png)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-04-08 19-44-56](https://user-images.githubusercontent.com/91142432/236634455-3d2efc42-4e12-4efb-9a75-e9e405d504b2.png)